cypress/e2e/settings/data-export/new-job-profile.cy.js

```diff
@@ -0,0 +1,86 @@
+import testTypes from "../../../support/dictionary/testTypes";
+import devTeams from "../../../support/dictionary/devTeams";
+import permissions from "../../../support/dictionary/permissions";
+import TopMenu from "../../../support/fragments/topMenu";
+import SettingsPane from "../../../support/fragments/settings/settingsPane";
+import JobProfiles from "../../../support/fragments/settings/data-export/jobProfiles";
+import getRandomPostfix from "../../../support/utils/stringTools";
+import Users from "../../../support/fragments/users/users";
+import FieldMappingProfiles from "../../../support/fragments/settings/data-export/fieldMappingProfiles";
+import InteractorsTools from "../../../support/utils/interactorsTools";
+
+let user;
+let newJobProfileName = `jobProfile${getRandomPostfix()}`
+let secondNewJobProfileName = `secondJobProfile${getRandomPostfix()}`
+let fieldMappingProfileName = `fieldMappingProfile${getRandomPostfix()}`;
+let newJobProfileCalloutMessage = `Job profile ${newJobProfileName} has been successfully created`
+let secondNewJobProfileCalloutMessage = `Job profile ${secondNewJobProfileName} has been successfully created`
+let newJobProfileDescription = `Decription${getRandomPostfix()}`
+
+describe('settings: data-export', () => {
+  before('create user, job and navigate to page', () => {
+    cy.createTempUser([
+      permissions.dataExportEnableSettings.gui,
+      permissions.dataExportEnableApp.gui,
+    ])
+      .then(userProperties => {
+        user = userProperties;
+        cy.login(user.username, user.password, { path: TopMenu.settingsPath, waiter: SettingsPane.waitLoading });
+      });
+
+    FieldMappingProfiles.goTofieldMappingProfilesTab();
```

Comment:
It's be better if you will create mapping profile by API

Reply:
done